src/routes/+page.svelte

```diff
@@ -1,2 +1,2 @@
-<h1>Welcome to SvelteKit</h1>
+<h1 class="text-lime-600">Welcome to SvelteKit</h1>
```

Comment:
This won't be an issue, but I removed this code.

I'll add the base HTML back in my working branch so it doesn't create a conflict.